### PR TITLE
Add feedback method

### PR DESCRIPTION
**Pull request by electrostat**
- add feedback creation method that references existing buildFeedbackEvent from telemetry library

### CODE REVIEW

File: libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationMetricsWrapper.java [hunk 1]
```diff
@@ -124,6 +124,54 @@ static void rerouteEvent(SessionState sessionState, RouteProgress routeProgress,
     ));
   }
 
+  static void feedbackEvent(SessionState sessionState, RouteProgress routeProgress, Location location,
+                            String description, String feedbackType, String screenshot) {
+    String upcomingInstruction = null;
```
Comment (Guardiola31337) [inline libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationMetricsWrapper.java]: The code from L129 to L156 is the same exact code implemented in `rerouteEvent` 👀 https://github.com/mapbox/mapbox-navigation-android/blob/e7ae10d673d61dc4a1e91d26ddb4de7ae3c1197a/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationMetricsWrapper.java#L72-L99
What about avoiding duplicate code extracting that block into a shared method?

File: libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationMetricsWrapper.java [hunk 2]
```diff
@@ -124,6 +124,54 @@ static void rerouteEvent(SessionState sessionState, RouteProgress routeProgress,
...
+      previousModifier = routeProgress.currentLegProgress().currentStep().getManeuver().getModifier();
+    }
+
+    Location[] beforeLocations = obtainLocations(sessionState.beforeRerouteLocations());
+
+    Location[] afterLocations = obtainLocations(sessionState.afterRerouteLocations());
+
+    String previousName = routeProgress.currentLegProgress().currentStep().getName();
+
+    MapboxNavigationEvent.buildFeedbackEvent(sdkIdentifier, BuildConfig.MAPBOX_NAVIGATION_VERSION_NAME,
```
Comment (Guardiola31337) [inline libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationMetricsWrapper.java]: This is currently doing nothing. We're missing pushing the event:

`MapboxTelemetry.getInstance().pushEvent(...);`

**Comment by danesfeder** [comment]
@electrostat looks like the build is failing from the MAS 3.0 changes - you just need to adjust those methods in `NavigationMetricsWrapper`

**Comment by electrostat** [comment]
@danesfeder yep, working on it right now.

**Comment by electrostat** [comment]
Need to review Feedback Event spec to compensate for changes.

**Comment by cammace** [comment]
Going ahead and merging this to unblock the feedback UI work. Any fixes needed can be made in a different branch.

**Pull request merged**